CHANGES.rst

```diff
@@ -21,6 +21,8 @@ Internationalization
 Improvements
 ^^^^^^^^^^^^
 
+- Include attachments when re-sending emails from the event log (:pr:`7203`, thanks
```

Comment:
New entries go to the bottom of the list, and I'd also reference the other PR that adds the new setting to store attached files here